app/utils/SQLHelpers.scala

```diff
@@ -58,32 +59,30 @@ class SimpleSQLDAO @Inject()(sqlClient: SQLClient)(implicit ec: ExecutionContext
              retryIfErrorContains: List[String] = List()): Fox[R] = {
     val foxFuture = sqlClient.db.run(query.asTry).map { result: Try[R] =>
       result match {
-        case Success(res) => {
+        case Success(res) =>
           Fox.successful(res)
-        }
-        case Failure(e: Throwable) => {
+        case Failure(e: Throwable) =>
           val msg = e.getMessage
           if (retryIfErrorContains.exists(msg.contains(_)) && retryCount > 0) {
-            logger.debug(s"Retrying SQL Query (${retryCount} remaining) due to ${msg}")
+            logger.debug(s"Retrying SQL Query ($retryCount remaining) due to $msg")
```

Comment:
I remember you said `debug` output of some instances is suppressed. Do we want to change the logging type here?

Reply:
If I remember correctly, that is a standalone-datastore-only issue, and debug should be fine here. It is certainly better in terms of semantics.